napari/components/_dims/model.py

```diff
@@ -125,9 +125,9 @@ def ndim(self):
     def ndim(self, ndim):
         if ndim > self.ndim:
             for i in range(self.ndim, ndim):
-                self._range.insert(0, (0.0, 1.0, 0.01))
-                self._point.insert(0, 0.0)
-                self._interval.insert(0, (0.3, 0.7))
+                self._range.insert(0, (0, 2, 1))
+                self._point.insert(0, 0)
+                self._interval.insert(0, (0, 1))
```

Comment:
this looks less crazy than what was there before, but it's still confusing to me! Would you mind adding comments about the insert syntax and what is happening for each of these? I particularly don't understand range vs interval...

Reply:
have added comments to clarify / improved doc strings